ee/clickhouse/sql/person.py

```diff
@@ -50,6 +51,7 @@
 team_id,
 properties,
 is_identified,
+deleted,
```

Comment:
same here - I'd make this `is_deleted` just to match the pattern we have above for `is_identified`